iree/tools/android/run_module_app/README.md

```diff
@@ -0,0 +1,62 @@
+# iree-run-module Android App
+
+This directory contains an Android application packing the `iree-run-module`
+command-line tool together with a specific IREE VM module invocation.
+
+Note that this app is **purely** for benchmarking/profiling IREE itself.
```

Comment:
Why is this better than adb-pushing binaries and assets?

Reply:
Because tools like [AGI](https://github.com/google/agi) requires a full Android app. That's basically the entire purpose of going through all the steps here. I would really love to just use the command-line version for this! But unfortunately..